pom.xml

```
@@ -4,7 +4,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.xerial</groupId>
   <artifactId>sqlite-jdbc</artifactId>
-  <version>3.34.0.1-SNAPSHOT</version>
+  <version>3.35.0</version>
```

```suggestion
  <version>3.35.0-SNAPSHOT</version>
```
We don't need to change this version before the release.